src/fides/api/alembic/migrations/versions/14acee6f5459_translation_data.py

```diff
@@ -0,0 +1,827 @@
+GB_REGIONS_TO_REMOVE = {
+    "gb_eng",
+    "gb_sct",
+    "gb_wls",
+    "gb_nir",
+}
+GB_REGIONS_TO_ADD = {
+    "gb",
+}
+
+
+def dump_table_to_csv(bind, table_name):
+    csv_file_path = f'./multilang_migration_backup_{table_name}_{datetime.now().strftime("%Y-%m-%d_%H-%M-%S")}.csv'
+    query = text(f"SELECT * FROM {table_name}")
+    result = bind.execute(query)
+
+    # Get column names from the result set
+    column_names = result.keys()
+
+    # Write results to CSV
+    with open(csv_file_path, "w") as csv_file:
+        csv_writer = csv.writer(csv_file)
+
+        # Write header row
+        csv_writer.writerow(column_names)
+
+        # Write data rows
+        for row in result:
+            csv_writer.writerow(row)
+
+
+def load_experience_config_from_files(file_paths):
+    """
+    Attempts loading the experience config from a set of given file paths until it finds a valid yaml
+    """
+    for path in file_paths:
+        try:
+            with open(path, "r") as f:
+                return yaml.safe_load(f).get("privacy_experience_configs")
+        except FileNotFoundError:
+            # If the file is not found in the current path, try the next one
+            continue
+
+    raise Exception("No privacy experience config file was found for migration!")
+
+
+def load_default_experience_configs():
+    """
+    Loads default experience config definitions from yml file.
+
+    Returns two dicts in a tuple:
+        - first dict holds "new" experience config records that will be used to reconcile existing data
+        - second dict holds the "raw" experience config records based on the definitions on disk. these will be used as a reference for e.g. template creation.
+    """
+    _raw_experience_config_map = (
+        {}
+    )  # this will just hold config templates exactly as they are in the 'oob' file
+    _reconciled_experience_config_map = (
+        {}
+    )  # will hold experience configs 'reconciled' between existing data and 'oob' template data
+
+    for experience_config in EXPERIENCE_CONFIG_DEFAULTS:
+        experience_config_type = DefaultExperienceConfigTypes(experience_config["id"])
+        _raw_experience_config_map[experience_config_type] = experience_config.copy()
+
+        # the reconciled experience config will have its regions populated by existing notices that are
+        # associated with this experience config. so in most cases, we start "fresh" with an empty set.
+        # the TCF experience and Canada banner modal experience are exceptions:
+        # their regions are not derived from notices (TCF is not linked to notices; Canada experience config is net-new),
+        # so we hardcode their regions to the config defaults.
+        reconciled_regions = (
+            experience_config["regions"]
+            if experience_config_type == DefaultExperienceConfigTypes.EEA_TCF_OVERLAY
+            or experience_config_type
+            == DefaultExperienceConfigTypes.CANADA_BANNER_MODAL
```

Comment:
At the end of this we have an EEA Banner and Modal with EEA regions and canadian regions and 4 notices, and a Canada Banner and Modal with canadian regions and no notices.

In my opinion the end state would not have canadian regions on the EEA banner and modal, and the Canada Banner and Modal would have the four notices.

As-is both cannot be enabled at once due to the overlapping regions.

Reply:
oh sorry yes I was not clear at all.